### DISCUSSION TITLE
Is there a possible performance/accuracy or other type of loss in converting from other frameworks to onnx?

### DISCUSSION THREAD

**Discussion by pedoc** [onnx/onnx#5397] [Q&A]
Hello everyone, thanks to onnx for allowing .neter to participate in AI.

As a beginner, I have a question
We know that the models generated by current mainstream frameworks such as tf pytorch can be easily converted to onnx format, so is there any distortion between the converted and the original model?

For example
1. Compared with the original model, will it take longer to load the onnx model?
2. Compared with the original model, will the onnx model reasoning take longer?
3. Compared with the original model, will the accuracy of onnx model recognition be lower?

On the contrary, in addition to bringing a unified format, is the onnx model better than other model formats?

Thank you for your time, thanks in advance.

**Comment by andife** [comment]
Maybe the talk from the last community meetup might be interesting for you: https://www.youtube.com/watch?v=Ks3rPKfiE-Y&list=PLNqLlwTfo322wVVTsH_ocKtSj8FEEnVpJ&index=3
